src/js/profile/mobile/widget/DropdownMenu.js

```diff
@@ -1051,13 +1052,22 @@
 				}
 
 				if (areaInfo.direction === "top") {
+					// check if we clicked on the element which is partially covered on the bottom of list
+					if (areaInfo.belowArea < 0) {
+						hiddenPart = areaInfo.belowArea;
+					}
 					offsetTop = self._offsetTop - height - scrollTop + ui.elPlaceHolder.offsetHeight;
 					ui.elOptionWrapper.classList.add(classes.top);
 				} else {
+					// check if we clicked on the element which is partially covered on the top of list
+					if (areaInfo.topArea < ui.elPlaceHolder.offsetHeight) {
+						hiddenPart = scrollTop % ui.elPlaceHolder.offsetHeight;
```

Comment:
Is it correct to calculate with Modulus?
Isn't it necessary to calculate the remaining value after calculation?

Reply:
yes, 
in example if you scroll up so the the second line is partially hidden and you click it
you have:
areaInfo.topArea = 36
scrollTop = 83
ui.elPlaceHolder.offsetHeight = 60

so in order to get proper hiddenPart you need take the rest

Reply:
I was checking possibility of using
`hiddenPart = ui.elPlaceHolder.offsetHeight - areaInfo.topArea;`
but this overlaps a header little bit
scrollTop is much higher than ui.elPlaceHolder.offsetHeight when element clicked on the second line that is why I use modulo here